### PR TITLE
add python Duplicate/Wrap implementation

### PR DESCRIPTION
**Pull request by whooo**
Fixes https://github.com/tpm2-software/tpm2-pytss/issues/145

### CODE REVIEW

File: tpm2_pytss/makecred.py [hunk 1]
```diff
@@ -96,3 +97,40 @@ def MakeCredential(public, credential, name):
     credblob = TPM2B_ID_OBJECT(credential=hmacdata + enc_cred)
     secret = TPM2B_ENCRYPTED_SECRET(secret=enc_seed)
     return (credblob, secret)
+
+
+def Wrap(newparent, public, sensitive, symkey, symdef):
```
Comment (williamcroberts) [inline tpm2_pytss/makecred.py:102]: I wish the C code was this clean, looks great and thank you.

**Review by williamcroberts — APPROVED** [review]
Assuming this all just passes in the CI, I have no comments.

File: tpm2_pytss/makecred.py (hunk 1 above)
Comment (whooo) [reply tpm2_pytss/makecred.py:102]: It's a lot easier when you don't have to throw pointers + sizes around :-)

**Comment by whooo** [comment]
Excluding tests for all the ESAPI calls, what's left for a release candidate?

**Comment by williamcroberts** [comment]
> Excluding tests for all the ESAPI calls, what's left for a release candidate?

https://github.com/tpm2-software/tpm2-pytss/milestone/3

But I think #88 is closed. Looks like intial support landed in dfbb821081

**Pull request merged**